static/tutorial/package-async.md

```diff
@@ -42,7 +42,11 @@ you can get away with `concurrently`/`race`/`Concurrently`, you should.
 
 ```haskell
 #!/usr/bin/env stack
--- stack --install-ghc --resolver lts-5.10 runghc --package async
+{-
+stack script
+  --resolver lts-8.21
+  --package async
```

Comment:
I like this change, but how about just leaving out the `--package async` entirely and let Stack parse the imports?